File: 268-missing-number/missing-number.php
class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function missingNumber($nums) {
        $count = count($nums);
        sort($nums);
        for($i=0; $i<=count($nums)-1 ; $i++){
            if($i != $nums[$i]){
                return $i ;
           /*  }else{
                return $count; */
            }
        }
        return $count;
    }
}

File: 268-missing-number/missing-number.dart
class Solution {
  int missingNumber(List<int> nums) {
    nums.sort();
    int res = 0;
    for(int i=0; i<nums.length;i++){
         if(nums[i] == i ){
            res++;
        }else{
            return res;
        }
    }
    return nums.length ;
  }
}